app/test_main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import pytest
+
+from unittest import mock
+
+from .main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "valid_url,connection,result",
+    [
+        (True, False, "Not accessible"),
```

Comment:
Add an error message. It's good when the user understands what the problem is. Fix everywhere

Reply:
> Add an error message. It's good when the user understands what the problem is. Fix everywhere

Do I understand this right, that you mean IDs? I have added them